### DISCUSSION TITLE
Suggestion: Competitors

### DISCUSSION THREAD

**Discussion by Guacomo88** [voxeltycoon/issues#363] [General]
Dear Dev. Team,

Your game looks and sounds great (I haven't bought it as yet), but I couldn`t find any informations about competitors KI.

This would be a strong feature for long term motivation.  I do not play Games like Cities Skylines, Transport Fever, and so on, due to the lack of competition challenges. I can remember the game TransportGigant from Jowood. It was amazing because of the KI competitors. Without it wouldn't have been that great!

With kind regards,
Guacomo

**Comment by JezzyBelle** [comment]
I don't know what you would call the mode the game is currently played in. I will call it Open World. I love Open World's because they are alive and generally encourage your own play style to solve the world's problems. So I love this mode and would hate for it to be removed. 

I could see Competitive mode, Multiplayer modes (Co-Op and PVP), and Creative Mode for working things out (maybe even have workshop blueprints). I would vote for it if it was an "additional" play mode, but just not the "default" game style/mode. But who knows... Maybe the Devs already decided this issue and just have not implemented their plans yet. 

Competitive mode is not for me generally unless it was casual or did not result in me having to quit because I was run out of business within one hour as a result. Competitive mode, to me, can become game breaking and will tend to shorter competitive game play where you are more concerned about expansion and exploitation rather than efficiencies and aesthetics as well as playing against your own goals.
